packages/datadog-plugin-couchbase/src/index.js

```diff
@@ -15,7 +15,7 @@ class CouchBasePlugin extends Plugin {
     this.addSub(`apm:couchbase:${func}:finish`, finish)
   }
 
-  startSpan (operation, customTags, store, { bucket, collection }) {
+  startSpan (operation, customTags, store, containers) {
```

Comment:
Since "bucket" and "collection" are fundamental concepts of Couchbase, I would keep them individually instead of creating a new term, even if older versions didn't have collections. Adding the concept of collection is enough to normalize the data published while keeping backward compatibility.